src/follower.py

```diff
@@ -66,11 +66,18 @@ def __init__(self):
         self._tag_lost_time = 0
         self._speed_cycle_time = time()
 
+        self.camera_angle_offset = 0
+
+        self.STRAIGHT_ANGLE = 90
+        self.WHEEL_MAX = 135
```

Comment:
This variable needs to be more descriptive. Wheel max what? The radius of the wheel itself? The angle it can turn? The speed it can spin at?